### ISSUE TITLE
network: Duplicate ids

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#15193]
On the Network page, these a11y issues should be fixed:

- [x] Logs have role-related issues, MM - this will be fixed as part of #15191 as it uses the same renderer
- [x] "Receiving" `<td>`s have `role="cell&quot;"`. It's odd that they have a role of "cell", but no matter what, shouldn't have the `&quot;`
- [x] Firewall switch lacks a label. Either it should have a `<label>` (with a for, or wrapping the widget), aria-label, or aria-labelledby. Additionally, it's our custom switch widget (`onoff-ct`) and should be ported to PF4's switch widget.

Firewall section:
- [ ] duplicate IDs — an ID should be unique on a page. Examples: expandable-toggle0, expandable-toggle2, etc. and expanded-content1, expanded-content3, etc. See https://github.com/patternfly/patternfly-react/issues/5690 https://github.com/cockpit-project/cockpit/pull/15956
    - example generated DOM: `<button aria-labelledby="simple-node0 expandable-toggle0" id="expandable-toggle0" aria-expanded="false" aria-disabled="false" aria-label="Details" class="pf-c-button pf-m-plain" type="button" data-ouia-component-type="PF4/Button" data-ouia-safe="true" data-ouia-component-id="OUIA-Generated-Button-plain-1">`
    - example DOM, part 2: `<td class="" colspan="4" data-key="0" id="expanded-content1">`
- [x] Firewall switch lacks a label. Either it should have a `<label>` (with a for, or wrapping the widget), aria-label, or aria-labelledby. Additionally, it's our custom switch widget (`onoff-ct`) and should be ported to PF4's switch widget. (Same as overview.)
